services/QuillLMS/engines/evidence/app/controllers/evidence/research/gen_ai/trials_controller.rb

```diff
@@ -93,6 +93,10 @@ def show
           end
         end
 
+        def llm_examples
```

Comment:
This is here due to the error with `LLMExamplesController`: `NameError:
  uninitialized constant Evidence::Research::GenAI::LLMExamplesController`. This is not RESTful, but works.